docs-devsite/vertexai-preview.vertexaierror.md

```diff
@@ -0,0 +1,76 @@
+Project: /docs/reference/js/_project.yaml
+Book: /docs/reference/_book.yaml
+page_type: reference
+
+{% comment %}
+DO NOT EDIT THIS FILE!
+This is generated by the JS SDK team, and any local changes will be
+overwritten. Changes should be made in the source code at
+https://github.com/firebase/firebase-js-sdk
+{% endcomment %}
+
+# VertexAIError class
+Error class for the Vertex AI for Firebase SDK.
+
+<b>Signature:</b>
+
+```typescript
+export declare class VertexAIError extends FirebaseError 
+```
+<b>Extends:</b> [FirebaseError](./util.firebaseerror.md#firebaseerror_class)
+
+## Constructors
+
+|  Constructor | Modifiers | Description |
+|  --- | --- | --- |
+|  [(constructor)(code, message, customErrorData)](./vertexai-preview.vertexaierror.md#vertexaierrorconstructor) |  | Creates a new VertexAIError instance. |
```

Comment:
Can VertexAIError be tagged in a way so that it displays as code font in the rendered ref docs?

Reply:
Fixed- I wrapped it in `<code>` [here](https://github.com/firebase/firebase-js-sdk/pull/8240/commits/a6b28b5bf21f63027ba8408f8dc3eeb5c35a415c?diff=unified&w=0#diff-b411e0790dceb6354c3392c8523db0cecff6cc6b010e9fc416018c640fa8286aR26), so it renders in the code font